apps/oracles/stock-price-feeds/src/fetch_prices.rs

```diff
@@ -80,7 +81,7 @@ pub async fn get_prices(
         fetch::<FMPPriceFetcher>(&symbols.fmp, get_api_keys(capabilities, &["FMP_API_KEY"])),
     ]);
 
-    let fetched_provider_prices = fetch_all_prices(futures_set).await;
+    let fetched_provider_prices = fetch_all_prices(futures_set, &Duration::from_secs(15)).await;
```

Comment:
Should we increase this timeout as well, the current voting period of all price feeds is 90 seconds, we could go for the same value as in cex-price-feeds might be?